pkg/cmd/taskrun/describe_test.go

```diff
@@ -731,3 +731,99 @@ func TestTaskRunWithSpecDescribe_custom_timeout(t *testing.T) {
 	}
 	golden.Assert(t, actual, fmt.Sprintf("%s.golden", t.Name()))
 }
+
+func TestTaskRunDescribe_last(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	/*
+		list.GetAllTaskRuns( ) returns a slice of strings containing all taskruns starting from the most recent one,
+		hence here the recent one is kept as the first element of the slice `trs`
+	*/
+	trs := []*v1alpha1.TaskRun{
```

Comment:
I would search for how tests for API version `v1beta1` in this file and model your tests off the way TaskRuns are included as test data in those. `v1alpha1` tests rely on test builders (i.e. `tb`), which are being moved away from.

Reply:
Sure